netcore/System.Private.CoreLib/src/System/Exception.cs

```diff
@@ -30,7 +31,7 @@ public DispatchState (MonoStackFrame[] stackFrames)
 		string _helpURL;
 		object _traceIPs;
 		string? _stackTraceString;
-		string? _unused3;
+		string? _remoteStackTraceString;
```

Comment:
SerializationRemoteStackTraceString needs updating too

Reply:
Thanks for fixing my mistake... (juggling couple of branches now)